test/testRunner.js

```diff
@@ -317,35 +346,273 @@ describe('JES explorer function verification tests', () => {
+
+                    expect(await testFilterDisplayStringValues(driver, expectedFilter)).to.be.true;
+                    expect(await testElementAppearsXTimesByCSS(driver, '.job-instance', 0)).to.be.true;
+                });
+            });
+            describe('JobId Url Filter', () => {
+                let jobIds;
+                before('get jobIds list from jobs filtered by IZUSVR owner', async () => {
+                    const filters = { owner: 'IZUSVR' };
+                    await loadUrlWithSearchFilters(driver, filters);
+                    const jobObjs = await waitForAndExtractParsedJobs(driver);
+                    jobIds = jobObjs.map(j => { return j.jobId; });
+                });
+
+                it('Should handle setting jobID from url query', async () => {
+                    const expectedJobId = jobIds[0];
+                    const filters = { jobId: expectedJobId };
+                    const expectedFilter = { ...DEFAULT_SEARCH_FILTERS, ...filters };
+
+                    await loadUrlWithSearchFilters(driver, filters);
+
+                    expect(await testFilterDisplayStringValues(driver, expectedFilter)).to.be.true;
+                    expect(await testJobIdFilter(driver, [expectedJobId])).to.be.true;
+                });
+            });
+
+            describe('Status Url Filter', () => {
+                it('Should handle setting status from url query');
+                it('Should handle fetching only ACTIVE jobs', async () => {
+                    const filters = { status: 'ACTIVE' };
+                    const expectedFilter = { ...DEFAULT_SEARCH_FILTERS, ...filters };
+                    const expectedStatus = ['ACTIVE'];
+
+                    await loadUrlWithSearchFilters(driver, filters);
+
+                    expect(await testFilterDisplayStringValues(driver, expectedFilter)).to.be.true;
+                    expect(await testJobStatusFilter(driver, expectedStatus)).to.be.true;
+                });
+
+                // TODO:: Can't gurantee we will have jobs in INPUT state so skip until we can
+                it.skip('Should handle fetching only INPUT jobs', async () => {
+                    const filters = { status: 'INPUT' };
+                    const expectedFilter = { ...DEFAULT_SEARCH_FILTERS, ...filters };
+                    const expectedStatus = ['INPUT'];
+
+                    await loadUrlWithSearchFilters(driver, filters);
+
+                    expect(await testFilterDisplayStringValues(driver, expectedFilter)).to.be.true;
+                    expect(await testJobStatusFilter(driver, expectedStatus)).to.be.true;
+                });
+
+                it('Should handle fetching only OUTPUT jobs', async () => {
+                    const filters = { status: 'OUTPUT' };
+                    const expectedFilter = { ...DEFAULT_SEARCH_FILTERS, ...filters };
+                    const expectedStatus = ['ABEND', 'OUTPUT', 'CC', 'CANCELED', 'JCL ERROR'];
```

Comment:
Another possible output status is SYS FAIL

Reply:
Added SYS status